exonum/src/blockchain/mod.rs

```diff
@@ -298,7 +299,13 @@ impl BlockchainMut {
 
         // Save & execute transactions.
         for (index, hash) in tx_hashes.iter().enumerate() {
-            self.execute_transaction(*hash, height, index, &mut fork, tx_cache);
+            self.execute_transaction(
+                *hash,
+                height,
+                index.try_into().unwrap(),
```

Comment:
Index is `usize` because its an index of `Vec` which has a `usize` type. And we don't make any conversions inside of `execute_transaction` method, cause signature of `execute_transaction` has been changed as well.

Reply:
Ah, sorry, I've accidentally opened a file from `master` instead of the actual one; nevermind.

But nevertheless, we can avoid `try_into().unwrap()` here as well, by using `zip`.

Reply:
Ok. Here it obviously makes sense 👍🏻